index.html

```diff
@@ -12,98 +12,264 @@
+          <ul class="project-langs">
+            <li class="project-lang">Ruby On Rails</li>
+            <li class="project-lang">CSS</li>
+            <li class="project-lang">JavaScript</li>
+            <li class="project-lang">HTML</li>
+          </ul>
+          <a href="#" class="button b3">See Project</a>
+        </li>
+        <li class="project">
+          <div class="project-img">
+            <img
+              class="img"
+              src="src/resources/Img_Plaholder.png"
+              alt="image placeholder"
+            />
+          </div>
+          <h3 class="project-title">Multi-Post Stories<br />Gain+Glory</h3>
+          <ul class="project-langs">
+            <li class="project-lang">Ruby On Rails</li>
+            <li class="project-lang">CSS</li>
+            <li class="project-lang">JavaScript</li>
+            <li class="project-lang">HTML</li>
+          </ul>
+          <a href="#" class="button b4">See Project</a>
+        </li>
+        <li class="project">
+          <div class="project-img">
+            <img
+              class="img"
+              src="src/resources/Img_Plaholder.png"
+              alt="image placeholder"
+            />
+          </div>
+          <h3 class="project-title">Multi-Post Stories<br />Gain+Glory</h3>
+          <ul class="project-langs">
+            <li class="project-lang">Ruby On Rails</li>
+            <li class="project-lang">CSS</li>
+            <li class="project-lang">JavaScript</li>
+            <li class="project-lang">HTML</li>
+          </ul>
+          <a href="#" class="button b5">See Project</a>
+        </li>
+        <li class="project">
+          <div class="project-img">
+            <img
+              class="img"
+              src="src/resources/Img_Plaholder.png"
+              alt="image placeholder"
+            />
+          </div>
+
+          <h3 class="project-title">Multi-Post Stories<br />Gain+Glory</h3>
+          <ul class="project-langs">
+            <li class="project-lang">Ruby On Rails</li>
+            <li class="project-lang">CSS</li>
+            <li class="project-lang">JavaScript</li>
+            <li class="project-lang">HTML</li>
+          </ul>
+          <a href="#" class="button b6">See Project</a>
+        </li>
+      </ul>
+    </section>
+    <section class="about-me">
+      <div class="about-me-container">
+        <h3 class="row1">About me</h3>
+        <p class="parag">
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to contact me.
+        </p>
+        <a href="#" class="button b7">Get my Resume</a>
       </div>
     </section>
-
+    <section class="experience-section">
+      <ul class="experience-ul">
+        <li class="experience">
+          <div class="logo">
+            <img
```

Comment:
- [ ] The technology categories ("Languages", "Frameworks" and "Skills") list belong to the about me section. Kindly move it there.